index.js

```diff
@@ -59,6 +60,11 @@ module.exports = function(source) {
     }
   });
 
+  manualImports.forEach(function(dep) {
+    var fullPath = utils.lookup(dep, options.paths, '', true);
+    self.addDependency(fullPath);
```

Comment:
Should be done as part of the block below. Maybe something like

```js
var boundResolvers = PathCache.resolvers(options, this.resolve);
var cache = manualImports.concat(options.filename).reduce(function(carry, filename) {
  return carry
    .then(function(lastCache) {
      return PathCache.createFromFile(boundResolvers, lastCache.contexts, filename);
    });
}, PathCache.create());

cache
  .then(function(importPathCache) {
```

This way any `@import` in `manualImports` files will work with webpack resolvers.